app/lang/en.json

```diff
@@ -134,5 +134,13 @@
 
     "certificateProviderDetails": "The certificate provider’s details",
     "certificateProviderDetailsHint": "Remember, the certificate provider must be 18 or over. They cannot be a member of your family (this includes step-relations and in-laws) or of the attorneys' families.",
-    "certificateProviderDobHint": "We need the certificate provider's date of birth to check they are over 18."
+    "certificateProviderDobHint": "We need the certificate provider's date of birth to check they are over 18.",
+
+    "aboutPayment": "About payment",
+    "aboutPaymentContent": "<p class=\"govuk-body\">Thank you for starting your application for an LPA.</p><p class=\"govuk-body\">It costs £82 to apply for an LPA. </p>",
```

Comment:
We have a mix of some pages having markup in the template and others in the translation file. I've kept the majority of markup in the translation file here but it would be good to get a consensus on this so it's consistent.

Reply:
I've been using HTML for anything that is obviously guidance or descriptive text over multiple paragraphs. A rule of thumb would be that it contains only and multiple govuk-heading/govuk-body/govuk-list/govuk-link elements. So for what you've got here I'd not insert the HTML for `feeCoverContent`, `lowIncomeWarning`, `qualifyingForReductionLink` and `continueToPaymentButton`, instead make them elements in the template and add translations for the inner text and labels.